test/cqlpy/test_secondary_index.py

```diff
@@ -376,6 +526,38 @@ def do_test(compression_opt, chunk_opt):
         for compression_opt, chunk_opt in itertools.product(compressor_opts, chunk_opts):
             do_test(compression_opt, chunk_opt)
 
+# Verify that we can set the compression property of an index, and that it will be successfully
+# applied to the underlying materialized view. That should be reflected in `system_schema.views`.
+def test_alter_index_compression(cql, test_keyspace, scylla_only):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY, v int, u int") as table:
+        index_name = unique_name()
+        view_name = f"{index_name}_index"
+
+        def do_test(compression_opt, chunk_opt):
+            opts = f"'sstable_compression': '{compression_opt}', 'chunk_length_in_kb': {chunk_opt}"
+
+            cql.execute(f"ALTER INDEX {test_keyspace}.{index_name} WITH compression = {{{opts}}}")
+
+            row_value = cql.execute(f"SELECT compression FROM system_schema.views WHERE " \
+                                    f"keyspace_name = '{test_keyspace}' AND view_name = '{view_name}'").one()
+
+            assert hasattr(row_value, "compression")
+            compression = row_value.compression
+
+            assert "sstable_compression" in compression
+            assert compression.get("sstable_compression") == compression_opt
+
+            assert "chunk_length_in_kb" in compression
+            assert compression.get("chunk_length_in_kb") == chunk_opt
+
+        cql.execute(f"CREATE INDEX {index_name} ON {table}(v)")
+
+        compressor_opts = ["org.apache.cassandra.io.compress.LZ4Compressor", "org.apache.cassandra.io.compress.SnappyCompressor"]
+        chunk_opts = ["4", "8"]
+
+        for compression_opt, chunk_opt in itertools.product(compressor_opts, chunk_opts):
+            do_test(compression_opt, chunk_opt)
+
```

Comment:
This patch adds even more hundreds of lines of tests for the properties of indexes - which makes me even more convinced it that all the new tests - probably close to 1000 lines of code - can be in one new file - maybe "test_secondary_index_properties.py" or something like that.